src/pybricks/hubs.pyi

```diff
@@ -38,4 +38,3 @@ class PrimeHub:
 
 
 class InventorHub(PrimeHub): ...
- 
```

Comment:
We should keep the new line at the end of the file. Many editors have a feature that does this automaticaly.

Reply:
There is a new line at the end of the file. Seems like GIT is confused because previously, there was a line with a single space.

IntelliJ deleted that extra space automatically keeping only the new line.

Reply:
Sounds good. GitHub was displaying a ⛔ earlier indicating that issue, but it seems gone now.

Reply:
It is back again:

![Screenshot from 2022-01-03 10-40-17](https://user-images.githubusercontent.com/12326241/147916778-fa73fdd4-3fbd-4cbc-9af2-15b87dccd095.png)

Reply:
This is GIT being confused I believe. If you check out the code, there is a new line at the end...

Reply:
I checked - it really isn't there:

![image](https://user-images.githubusercontent.com/12326241/147917769-f1e5dea6-2eae-49a4-8183-dee80fa51bb5.png)

Reply:
Previously there was `0A20` at the end of the file. Now there is `0A`only.

Reply:
Now I understand it! Previously, there was `0A20` which means there was no newline at the end of the file.

The ⛔ sign means the `missing new line` was removed, meaning there is a new line now 😆

Reply:
In my Git client, I do see that a `space` was removed from the last line
<img width="796" alt="Newline" src="https://user-images.githubusercontent.com/1579300/147918924-db9073a8-7f0d-4326-92e5-6e7f118ba6b5.png">